src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/CollectSVEvidence.java

```diff
@@ -764,4 +847,160 @@ public VariantContext next() {
+            } else {
+                final DepthEvidenceCodec codec = new DepthEvidenceCodec();
+                if ( !codec.canDecode(outputFilename) ) {
+                    throw new UserException("Attempting to write depth evidence to a file that " +
+                            "can't be read as depth evidence: " + outputFilename + ".  The file " +
+                            "name should end with \".rd.txt\", \".rd.txt.gz\", or \".rd.bci\".");
+                }
+                writer = codec.makeSink(outputDepthEvidencePath, dict, sampleNames, cmprLevel);
+            }
+
+            final FeatureDataSource<Feature> intervalSource =
+                    new FeatureDataSource<>(inputIntervalsPath.toPath().toString());
+            this.intervalIterator = intervalSource.iterator();
+            if ( !intervalIterator.hasNext() ) {
+                throw new UserException(inputIntervalsPath + " contains no intervals.");
+            }
+            this.minMapQ = minMapQ;
+            depthEvidence = new DepthEvidence(intervalIterator.next(), new int[1]);
+            summmedIntervalLengths += depthEvidence.getLengthOnReference();
+            nIntervals += 1;
+        }
+
+        void apply( final GATKRead read ) {
+            if ( read.getMappingQuality() < minMapQ ) {
+                return;
+            }
+            while ( depthEvidence != null ) {
+                final int cmp = lComp.compareLocus(read.getContig(), read.getStart(), depthEvidence);
+                if ( cmp < 0 ) { // if read is upstream of interval of interest, nothing to do
+                    return;
+                }
+                if ( cmp == 0 ) { // if read is in the interval of interest, count it
+                    depthEvidence.getCounts()[0] += 1;
+                    return;
+                }
+
+                // read is downstream of interval
+                writer.write(depthEvidence);
+                countCounter.addCount(depthEvidence.getCounts()[0]);
+                if ( !intervalIterator.hasNext() ) {
+                    depthEvidence = null;
+                    return;
+                }
+                depthEvidence = new DepthEvidence(intervalIterator.next(), new int[1]);
+                summmedIntervalLengths += depthEvidence.getLengthOnReference();
+                nIntervals += 1;
+            }
+        }
+
+        void close() {
+            if ( depthEvidence != null ) {
+                writer.write(depthEvidence);
+                countCounter.addCount(depthEvidence.getCounts()[0]);
+            }
+            writer.close();
+        }
+
+        void reportSummaryStats( final GATKPath summaryPath, final String sampleName ) {
+            try ( final BufferedWriter writer
+                     = new BufferedWriter(new OutputStreamWriter(summaryPath.getOutputStream())) ) {
+                final int[] quartiles = countCounter.getQuartiles();
+                writer.write("rd_q25_" + sampleName + "\t" + quartiles[0]);
```

Comment:
```suggestion
                writer.write("rd_q25\t" + sampleName + "\t" + quartiles[0]);
```
This would make aggregating metrics across samples a little easier